### ISSUE TITLE
Support for pasting from key

### ISSUE THREAD

**Issue by zDEFz** [ckb-next/ckb-next#871]
## What is the program's current behaviour?
Text Macro can be only assigned to a G key when user has typed it manually
## What would you like it to do instead?
Text macro can be pasted and on release is immediately entered

**Comment by grizzlysmit** [comment]
yes I want this too especially converting text to keystrokes so we can just pate it in
